Makefile

```diff
@@ -179,6 +179,18 @@ code/lint:
 	@echo "--> Running golangci-lint"
 	@$(shell go env GOPATH)/bin/golangci-lint run --timeout 10m
 
+.PHONY: client/gen
+client/gen:
+	@echo "--> Running code-generator to generate clients"
```

Comment:
@pskopek @ASzc Could you please have a look at this? Is this acceptable for our pipelines?

Reply:
I think it won't be a problem since it is not run by default (correct?) and therefore won't be ever executed in the pipelines.

Reply:
But this brings me to another question. Should we hook up this target with `code/gen`? I'm a bit afraid it'll get forgotten and we end up with outdated client...

Reply:
Gmail relegated this to the Forums tab for me (a tab with very high noise ratio), so I never saw it. Doing code generation in the pipelines is a big problem, as we've seen with operator-sdk. It really should just be a rule that this is run by the developers for each change, before handing off the code. If the code isn't right, that's not the build pipeline's problem.